src/functionalTest/java/uk/gov/hmcts/reform/judicialapi/JudicialUsersFunctionalTest.java

```diff
@@ -143,15 +133,7 @@ void refreshUserProfileSortDesc(String role) {
         Response refreshResponse = judicialApiClient.refreshUserProfiles(refreshRoleRequest, 1, 0,
                 "objectId", "DESC",role);
 
-        if (OK.value() == refreshResponse.getStatusCode()) {
-            List<UserProfileRefreshResponse> userProfiles = asList(refreshResponse.getBody()
-                    .as(UserProfileRefreshResponse[].class));
-
-            log.info("JRD get refreshResponse response: {}", userProfiles.get(0).getObjectId());
-            assertNotNull(userProfiles.get(0).getObjectId());
-        } else {
-            assertEquals(NOT_FOUND.value(),refreshResponse.getStatusCode());
-        }
+        assertEquals(BAD_REQUEST.value(),refreshResponse.getStatusCode());
```

Comment:
Looks like you have removed a success scenario and replaced with negative scenario. Now there are no success scenarios for refresh api. Can you add this as a separate test ?

Reply:
We can add success scenario but that would create dependency on database, if anybody update data or remove data in the database then our test case will start failing.

Reply:
that's why we have added these
`if (OK.value() == refreshResponse.getStatusCode()) {
            List<UserProfileRefreshResponse> userProfiles = asList(refreshResponse.getBody()
                    .as(UserProfileRefreshResponse[].class));
            log.info("JRD get refreshResponse response: {}", userProfiles.get(0).getObjectId());
            assertNotNull(userProfiles.get(0).getObjectId());
        } else {
            assertEquals(NOT_FOUND.value(),refreshResponse.getStatusCode());
        }`

Reply:
Yeah It make sense, I have added one success scenario as well. Thanks for the clarification.